### PR TITLE
Changing the Template Project to enable Application Framework, address HighDpiSettings and be compatible with net6.0.

### PR DESCRIPTION
**Pull request by KlausLoeffelmann**
This project resembles the updated VB Project which should be created by the upcoming template.

Note: This is NOT the template. Just the proposed and discussed result of the template, which we are working on right now.

Please restrict comments/questions to the immediate context of the template. Thanks!

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/3861)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.vb [hunk 1]
```diff
@@ -13,20 +13,20 @@ Option Explicit On
...
+        
+        <Global.System.Diagnostics.DebuggerStepThroughAttribute()>  _
         Public Sub New()
             MyBase.New(Global.Microsoft.VisualBasic.ApplicationServices.AuthenticationMode.Windows)
-            Me.IsSingleInstance = True
-            Me.EnableVisualStyles = True
-            Me.SaveMySettingsOnExit = False
+            Me.IsSingleInstance = false
+            Me.EnableVisualStyles = true
+            Me.SaveMySettingsOnExit = true
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.vb:29]: Should these be capitalised?

Comment (KlausLoeffelmann) [reply src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.vb:29]: Good catch! 
They have been generated by the App Designer (working for 16.8 Preview 4). 
Pinging @melytc - could you check if the App Designer for the Application Frameworks writes those in small caps?

File: src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.HighDpi.vb [hunk 2]
```diff
@@ -42,9 +56,24 @@ Namespace My
...
+
+#Else
+
+Friend Module Program
+    <STAThread()>
+    Friend Sub Main(args As String())
+        Application.SetHighDpiMode(HighDpiMode.SystemAware)
+        Application.EnableVisualStyles()
+        Application.SetCompatibleTextRenderingDefault(False)
+        Application.Run(New Form1)
```
Comment (paul1956) [inline src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.HighDpi.vb:74]: Doesn't Args need to be passed to Form as it is in Framework?

Comment (KlausLoeffelmann) [reply src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.HighDpi.vb:74]: > as it is in Framework?

Not sure what you mean, @paul1956. Where do you see this in Framework?

Comment (paul1956) [reply src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.HighDpi.vb:77]:
```
#Region "Assembly Microsoft.VisualBasic, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
' C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Microsoft.VisualBasic.dll
#End Region
Namespace Microsoft.VisualBasic.ApplicationServices
       '
       ' Summary:
       '     Sets up and starts the Visual Basic Application model.
       '
       ' Parameters:
       '   commandLine:
       '     Array of type String. The command line from Sub Main.
       <SecuritySafeCritical>
       Public Sub Run(commandLine() As String)
```

Comment (KlausLoeffelmann) [reply src/System.Windows.Forms/tests/IntegrationTests/VisualBasicTemplate/My Project/Application.Designer.HighDpi.vb:77]: We decided to pull the Module code `Sub Main` from Application.Designer.HighDpi.vb, so, this is no longer relevant.

**Comment by KlausLoeffelmann** [comment]
Templates are in based on this.

**Pull request closed without merging**